packages/components/src/components/icon/rebranding/ic-rebranding-deriv-trader.svg

```diff
@@ -0,0 +1 @@
M249.299 49.088c.693 0 1.354-.016 1.984-.047a13.81 13.81 0 0 0 1.512-.142V43.56a12.981 12.981 0 0 0-1.276-.189 14.196 14.196 0 0 0-1.559-.095c-.661 0-1.291.048-1.89.142a4.456 4.456 0 0 0-1.512.425c-.441.221-.787.52-1.039.898-.252.378-.378.85-.378 1.418 0 1.102.362 1.874 1.086 2.315.756.41 1.78.614 3.072.614Zm-.567-20.6c2.079 0 3.811.236 5.197.708 1.386.473 2.489 1.15 3.308 2.032.85.882 1.449 1.953 1.795 3.213.347 1.26.52 2.661.52 4.205v14.648c-1.008.22-2.41.472-4.205.756-1.796.315-3.969.472-6.521.472-1.606 0-3.071-.142-4.394-.425-1.292-.284-2.41-.74-3.355-1.37a6.555 6.555 0 0 1-2.173-2.552c-.504-1.04-.756-2.315-.756-3.827 0-1.45.283-2.678.85-3.686a6.926 6.926 0 0 1 2.363-2.41c.976-.598 2.094-1.023 3.354-1.275a17.82 17.82 0 0 1 3.922-.425c.914 0 1.717.047 2.41.141.724.063 1.307.158 1.748.284v-.662c0-1.197-.362-2.157-1.087-2.882-.724-.724-1.984-1.087-3.78-1.087-1.197 0-2.378.095-3.543.284-1.166.157-2.174.393-3.024.709l-.898-5.67c.409-.127.913-.253 1.512-.378.63-.158 1.307-.284 2.032-.378.724-.127 1.48-.221 2.268-.284a21.418 21.418 0 0 1 2.457-.142ZM269.556 41.386c0 2.174.489 3.922 1.465 5.245.977 1.323 2.426 1.985 4.347 1.985.63 0 1.213-.016 1.748-.047.536-.064.977-.127 1.323-.19V35.575c-.441-.284-1.023-.52-1.748-.709a8.046 8.046 0 0 0-2.126-.283c-3.339 0-5.009 2.267-5.009 6.803ZM285.48 53.2c-.63.189-1.355.378-2.174.567-.819.157-1.685.3-2.599.425-.882.126-1.795.22-2.74.284-.914.094-1.796.142-2.646.142-2.048 0-3.875-.3-5.481-.898-1.607-.599-2.961-1.45-4.064-2.552-1.102-1.134-1.953-2.488-2.551-4.063-.567-1.607-.851-3.402-.851-5.387 0-2.016.252-3.827.756-5.434.504-1.638 1.229-3.023 2.174-4.157a9.352 9.352 0 0 1 3.449-2.6c1.386-.598 2.961-.897 4.725-.897.977 0 1.843.095 2.599.284.787.189 1.575.456 2.362.803V18.47l7.041-1.134V53.2ZM288.528 41.764c0-2.205.331-4.126.992-5.764.693-1.67 1.591-3.056 2.694-4.158a11.153 11.153 0 0 1 3.78-2.504 12.09 12.09 0 0 1 4.441-.85c3.528 0 6.316 1.086 8.363 3.26 2.048 2.141 3.072 5.307 3.072 9.497 0 .41-.016.866-.048 1.37a78.32 78.32 0 0 1-.094 1.276h-15.971c.158 1.449.835 2.599 2.032 3.449 1.197.85 2.804 1.276 4.82 1.276 1.291 0 2.551-.11 3.78-.331 1.26-.252 2.283-.551 3.071-.898l.945 5.718c-.378.189-.882.378-1.512.567-.63.189-1.339.346-2.126.472-.756.157-1.575.284-2.457.378a24.82 24.82 0 0 1-2.646.142c-2.237 0-4.19-.331-5.859-.992-1.638-.662-3.009-1.56-4.111-2.694-1.071-1.165-1.874-2.535-2.41-4.11-.504-1.576-.756-3.276-.756-5.104Zm16.538-2.693a6.621 6.621 0 0 0-.331-1.748 3.903 3.903 0 0 0-.803-1.512c-.347-.441-.804-.803-1.371-1.087-.535-.283-1.212-.425-2.031-.425-.788 0-1.465.142-2.032.425a3.949 3.949 0 0 0-1.418 1.04c-.378.44-.677.96-.897 1.559-.189.567-.331 1.15-.426 1.748h9.309ZM329.918 35.386c-.63-.158-1.37-.315-2.22-.473a12.623 12.623 0 0 0-2.741-.283c-.441 0-.976.047-1.606.142-.599.063-1.056.141-1.371.236v18.994h-7.04v-23.53c1.26-.441 2.741-.85 4.442-1.229 1.732-.41 3.654-.614 5.764-.614.378 0 .835.031 1.37.095a13.96 13.96 0 0 1 1.607.189 14.3 14.3 0 0 1 1.606.283c.536.095.993.22 1.371.378l-1.182 5.812Z"/></svg>
```

Comment:
Also, some icons have extra empty space within SVG file, is there a way and a point to improve it?

Reply:
It is caused by the minifer.